README.md

```diff
@@ -471,6 +472,78 @@ properly set the `--shm-size` flag depending on the size of your inputs and
 outputs. The default value for docker run command is `64MB` which is very
 small.
 
+# Business Logic Scripting (beta)
+
+Triton's
+[ensemble](https://github.com/triton-inference-server/server/blob/main/docs/architecture.md#ensemble-models)
+feature supports many use cases where multiple models are composed into a
+pipeline (or more generally a DAG, directed acyclic graph). However, there are
+many other use cases that are not supported because as part of the model
+pipeline they require loops, conditionals (if-then-else), data-dependent
+control-flow and other custom logic to be intermixed with model execution. We
+call this combination of custom logic and model executions a Business Logic
+Scripting (BLS). 
+
+Starting from 21.08, you can execute inference requests on any other model that
```

Comment:
"Starting from 21.08, you can implement BLS in your Python model. A new set of utility functions allows you to execute inference requests on other models being served by Triton as a part of executing your Python model.